### DISCUSSION TITLE
Q3| Community Manager Proposal

### DISCUSSION THREAD

**Discussion by Cindy884** [ringdao-community-guild/discussion#21] [Show and tell]
# 1. Job Description

My work has been more specific over last month's practice and discussion. As a community manager, my role is to link the DCDAO and Darwinia. Based on this understanding, it can be divided into three parts as the following:

## 1.1 DCDAO Development

1.1.1 Establish DCDAO governance framework, including constitution discussion, voting system, etc
1.1.2 Promote DCDAO, and introduce more members to DCDAO
1.1.3 Operate DCDAO Social media account

## 1.2 Resources Allocation

In this part, my main work is to leverage the community resources to help the development of DCDAO and Darwinia. The resources include human resources and funding and so on.
## 1.3 Teams Collaboration

Stay closely connected with community members and Darwinia teams, answer their questions, address their concerns, and establish strong relationships.

# 2. Objective

## 2.1 Update Darwinia’s Image

Key Action
-Create the education tutorial with the DCDAO members together, including Youtube video, medium article

-Interact with members through these materials and gain more exposure to social media

## 2.2 Building DCDAO Ambassador Teams and Collaboration

Key Action

-Interview and select core members 

-Create the team’s workspace in dework and collaborate with team members

-Review the ambassador’s monthly report and send the rewards 

## 2.3 Gain the first 100 followers on Twitter account

Key Action

-Create valuable content and post it on Twitter

-Collect and retweet Darwinia news

-Interact with other Twitter users and gain more exposure

-Create some events like giveaway once a month 

## 2.4 **Teams Collaboration**

Key Action

-Organize DCDAO weekly Meetings and update the latest progress with DCDAO’s member

-Organize DCDAO Ambassador Monthly Meeting.

-Answer members’ daily questions

## 2.5 Others

-Bring more business partners to enrich the Darwina ecosystem

-Attract more users to be our ambassadors or contributors in the world, especially in Southeast Asia, Latin America, Turkey, etc

-Encourage community members to participate in their local web3.0 events

# 3. Funding Support

The funding support is approximately between 5000USDT/month. Here are the details.

## 3.1 Ambassador Program Monthly Expense

Within 2000u-3000USDT/month(the rewards will be paid by Ring)

## 3.2 Community Manager Monthly Salary

1700 USD(USDT is preferred)

My metamaskl wallet address: 0xc2B59Ecd40b8B54Bc23Bc395EefA0DEC3CCC6c5F

## 3.3 Other Expenses

In order to activate our community and promote DCDAO, I plan to do some events like giveaway or other types of interaction with members in Q3. In this part, the budget is within 1000 USDT （will be paid by Ring) quarterly.

**Comment by hackfisher** [comment]
Suggestions:

Section 2.5, Add monthly report for the position of community manager.

Section 3 only keeps 3.2, for section 3.1 and 3.3, suggest to move to another Update Proposal for https://snapshot.org/#/darwiniacommunitydao.eth/proposal/0x69bc76854f78a4d995b6ae47d27c95e12424c5aab60261238757575daa04e61d

**Comment by Cindy884** [comment]
My new receive wallet: 0xf11D5D94d41ffd0339104A74d3E33f808A27c54E（polygon is preferred)

**Reply by Cindy884** [reply]
My new receive wallet（multi-sig wallet): 
eth:0x7C548fe9aEd9F6C50cD372196eA5510ac56dBa8F
My new metamask wallet address:
0x0f9B66CB13f9f429C352005AB2b1255e2Dbe4Ab8
